reproject/mosaicking/wcs_helpers.py

```diff
@@ -44,7 +52,15 @@ def find_optimal_celestial_wcs(
             * A tuple where the first element is a `~numpy.ndarray` and the
               second element is either a `~astropy.wcs.WCS` or a
               `~astropy.io.fits.Header` object
+            * An `~astropy.nddata.NDData` object from which the ``.data`` and
+              ``.wcs`` attributes will be used as the input data.
 
+        If only one input data needs to be provided, it is also possible to
+        pass it in without including it in an iterable.
```

Comment:
Is the added complexity really worth this?

Reply:
Absolutely 😄